src/main/java/org/bihealth/mi/easysmpc/cli/User.java

```diff
@@ -0,0 +1,468 @@
+/* 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bihealth.mi.easysmpc.cli;
+
+import java.io.File;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.concurrent.FutureTask;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.bihealth.mi.easybus.BusException;
+import org.bihealth.mi.easybus.MessageListener;
+import org.bihealth.mi.easybus.Scope;
+import org.bihealth.mi.easybus.implementations.email.ConnectionIMAPSettings;
+import org.bihealth.mi.easysmpc.dataexport.ExportFile;
+import org.bihealth.mi.easysmpc.dataimport.ImportClipboard;
+import org.bihealth.mi.easysmpc.resources.Resources;
+
+import de.tu_darmstadt.cbs.emailsmpc.Bin;
+import de.tu_darmstadt.cbs.emailsmpc.BinResult;
+import de.tu_darmstadt.cbs.emailsmpc.Message;
+import de.tu_darmstadt.cbs.emailsmpc.Study;
+import de.tu_darmstadt.cbs.emailsmpc.Study.StudyState;
+
+/**
+ * A user in an EasySMPC process
+ * 
+ * @author Felix Wirth
+ * @author Fabian Prasser
+ */
+public class User implements MessageListener {
```

Comment:
Something is weird with the naming of this class. It is called user but often referred to as a process (e.g shutdown(), isProcessFinished). Maybe it needs to be renamed or redesigned? Cal it "UserProcess"? Split user-attributes and methods from process-attributes and methods?

Reply:
I can not follow. Please advice what to do

Reply:
Renamed to "UserProcess"